src/output/plugins/httpd/HttpdClient.cxx

```diff
@@ -102,6 +115,15 @@ HttpdClient::HandleLine(const char *line) noexcept
 			return true;
 		}
 
+
+		#ifdef HAVE_BASE64
```

Comment:
Preprocessor directives should not be indented

Reply:
ok.